msg/Surface.msg

```diff
@@ -0,0 +1,3 @@
+# Information of a collision surface for collision object construction
```

Comment:
Again, the purpose of the `Surface` is just to represent a surface. It may be used as a collision object, but it could also be used for visualization. So this comment should just say:

`# Represents a surface found when detecting surface objects.`